### ISSUE TITLE
RewardItem15' is marked as required, but was missing from the buffer.

### ISSUE THREAD

**Issue by TJP3-Github** [LegoFigure11/RaidCrawler#150]
**Bug**
This bug occurs when RaidCrawler v1.3.2 attempts to read Event Raid status, since start of the Chest Form Gimmighoul event.

**To Reproduce**
1. Launch SV in CFW Switch
2. Connect to Internet in-game to receive the latest event
3. Connect RaidCrawler to CFW Switch
4. Error occurs

**Expected behavior**
Normal operating behavior.

**Raid Data**
Unable to dump Raid Data.

**Screenshots**
![RaidCrawler Error](https://github.com/LegoFigure11/RaidCrawler/assets/66980268/0711521a-4622-4d57-93b4-661b4c7ee82c)

**Additional context**
Began at 8AM PT

**Comment by CakeFlour** [comment]
This is an issue exclusive to Scarlet Version because of Gimmighoul Event Raids having broken rewards.

**Comment by TJP3-Github** [comment]
Incorrect. This Bug is occurring in both my Scarlet & Violet versions.

**Comment by DuknBag** [comment]
And this is still present in the patched Scarlet Den

**Comment by DuknBag** [comment]
Thanks, that got it working

**Comment by TJP3-Github** [comment]
Deleting Cache folder fixed this issue in Violet for me, but not Scarlet.

**Comment by TJP3-Github** [comment]
Got it, yes post-hotfix and deleting the Cache Folder the issue has now gone away.

**Comment by Koi-3088** [comment]
Not closing yet since GF's ~~bugs~~ features should be handled by RC.
